octopoes/tests/integration/test_https_availability_nibble.py

```diff
@@ -11,14 +12,61 @@
 from octopoes.models.ooi.network import IPAddressV4, IPPort, Network, Protocol
 from octopoes.models.ooi.service import IPService, Service
 from octopoes.models.ooi.web import HostnameHTTPURL, HTTPHeader, HTTPResource, WebScheme, Website
+from octopoes.repositories.ooi_repository import XTDBOOIRepository
 
 if os.environ.get("CI") != "1":
     pytest.skip("Needs XTDB multinode container.", allow_module_level=True)
 
-STATIC_IP = ".".join((4 * "1 ").split())
+STATIC_IP = ".".join(4 * ["1"])
```

Comment:
Could we just write 1.1.1.1 here?

Reply:
The reason we write it that way is not to have SonarCloud report a "static ip security vulnerability".